src/panel-manager.js

```diff
@@ -9,6 +9,14 @@ genotet.panelManager = {};
 genotet.panelManager.COLLAPSED_WIDTH = 32;
 /** @const {number} */
 genotet.panelManager.TRANSITION_TIME = 300;
+/** @const {number} */
```

Comment:
```js
/** @private @const {number} */
genotet.panelManager.TRANSITION_TIME_ = 300;
```
Because it seems that these variables won't be used outside this file.